pyuvdata/uvdata/aipy_extracts.py

```diff
@@ -652,6 +653,12 @@ def init_from_uv(self, uv, override={}, exclude=[]):
         exclude : list
             list of variable to exclude
         """
+        if override is None:
+            override = {}
```

Comment:
I prefer `override = override or {}` but I'll leave that up to you.

Reply:
I think I'm not as good at reading python as you are. I wouldn't know what that meant if I encountered it.